controllers/provider-aws/pkg/webhook/controlplane/ensurer.go

```diff
@@ -205,3 +204,42 @@ func (e *ensurer) EnsureKubernetesGeneralConfiguration(ctx context.Context, data
 	*data = buf.String()
 	return nil
 }
+
+func appendUniqueUnit(units *[]extensionsv1alpha1.Unit, unit extensionsv1alpha1.Unit) {
```

Comment:
Does it make sense to move this to https://github.com/gardener/gardener-extensions/blob/master/pkg/webhook/controlplane/utils.go?